Gruntfile.cjs

```diff
@@ -209,6 +209,14 @@ module.exports = function( grunt ) {
 		grunt.log.writeln( "Node.js 17 or newer detected, skipping jsdom tests..." );
 	} );
 
+	grunt.registerTask( "authors", async function() {
+		const done = this.async();
+		const { getAuthors } = require( "./build/release/authors.js" );
+		const authors = await getAuthors();
+		console.log( authors.join( "\n" ) );
+		done();
```

Comment:
I like this pattern of using an async function and calling `done()` in a way that looks synchronous. 🙂